R/GetMethods.R

```diff
@@ -544,16 +544,20 @@ setMethod("filteredCTSSidx", "CAGEexp", function (object){
 #' @param object A [`CAGEr`] object.
 #' 
 #' @param sample Optional. Label of the CAGE dataset (experiment, sample) for
-#'        which to extract sample-specific information on consensus clusters.
+#'        which to extract sample-specific information on consensus clusters. 
+#'        When no sample is specified (NULL), sample-agnostic information 
```

Comment:
“sample-agnostic” is a bit of a paradox, because all samples are used to compute the IQ ranges in that case. How about “sample-averaged” or something else?  Also, to remove possible doubts, how about documenting somewhere that the normalised expression values are used, and not the raw counts?

Reply:
How about phrasing it like this: "… the same information on consensus clusters independent of sample-specific tag cluster boundaries ...”
#'        Label of the CAGE dataset (experiment, sample) for
#'        which to extract sample-specific information on consensus clusters. 
#'        When no sample is specified (NULL), _the same information, independent 
#'        of sample-specific tag cluster boundaries_, for 
#'        consensus clusters is provided. This includes the `dominant_ctss` 
#'        and `tpm.dominant_ctss` for each consensus cluster.

Reply:
Once we resolve this usage, I will update the vignette accordingly.